From 99cb28c225c625eb5c3f987971574e767f98f67c Mon Sep 17 00:00:00 2001
From: Wyon Bi <bivvy.bi@rock-chips.com>
Date: Thu, 22 Feb 2018 10:05:19 +0800
Subject: [PATCH] arm64: dts: rockchip: allow to disable vcc3v3_lcd on px30-evb
 board

remove the 'regulator-always-on' property from vcc3v3_lcd

Change-Id: I056601ed575b993065c17d245195bd376808bbed
Signed-off-by: Wyon Bi <bivvy.bi@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts | 3 +--
 arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts | 3 +--
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts b/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
index 1ca9a099c332..93acddb1d032 100644
--- a/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/px30-evb-ddr3-v10.dts
@@ -158,7 +158,7 @@
 	panel@0 {
 		compatible = "sitronix,st7703", "simple-panel-dsi";
 		reg = <0>;
-		//power-supply = <&vcc3v3_lcd>;
+		power-supply = <&vcc3v3_lcd>;
 		backlight = <&backlight>;
 		prepare-delay-ms = <2>;
 		reset-delay-ms = <1>;
@@ -486,7 +486,6 @@
 			};
 
 			vcc3v3_lcd: SWITCH_REG1 {
-				regulator-always-on;
 				regulator-boot-on;
 				regulator-name = "vcc3v3_lcd";
 			};
diff --git a/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts b/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts
index 1d40733013ce..0db2c6a6de4d 100644
--- a/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/px30-evb-ddr4-v10.dts
@@ -158,7 +158,7 @@
 	panel@0 {
 		compatible = "sitronix,st7703", "simple-panel-dsi";
 		reg = <0>;
-		//power-supply = <&vcc3v3_lcd>;
+		power-supply = <&vcc3v3_lcd>;
 		backlight = <&backlight>;
 		prepare-delay-ms = <2>;
 		reset-delay-ms = <1>;
@@ -487,7 +487,6 @@
 			};
 
 			vcc3v3_lcd: SWITCH_REG1 {
-				regulator-always-on;
 				regulator-boot-on;
 				regulator-name = "vcc3v3_lcd";
 			};
-- 
2.35.3

